Update News.md (#8257)
Microsoft/AL: News.md

# 2026-05-11 Copilot in Business Central moves to GPT-5.3-chat

From May through June, Copilot in Business Central is upgrading to GPT-5.3-chat as the default model. This update rolls out gradually to environments across all regions.

**What's changing:**

- **New default model** — GPT-5.3-chat replaces the previous model, bringing improved quality and performance for Copilot features in Business Central.
- **Model transparency for custom agents** — A new UI setting lets you see which model your custom agents are currently using and select your preferred model.

> - **Model control for SDK agents** — New methods in the agent SDK let you control model selection programmatically from code.

Reviewer: I don't think we externally talk about SDK?

> - **Model control for SDK agents** — New methods in the agent SDK let you control model selection programmatically from code.

Reviewer: ```suggestion
- **Model control for coded agents** — New methods when coding agents let you control model selection programmatically from AL.
```

**What to expect during rollout:**

The model update and the new UI for model selection roll out independently. Your environment might receive GPT-5.3-chat before the model selection UI becomes available. During this period, Copilot continues to work as expected with the new model.

Environments in the UK, India, and Australia will receive the update at a later date.

No action is required on your part. If you want to switch models for your custom agents after the selection UI becomes available, you can do so from the **Agents** page in Business Central. For SDK agents, use the new model selection methods to control which model your agent uses.

> 

Reviewer: For those of you having started on the journey of designing and coding agents, then we highly recommend evaluating your agents again once the model has updated, as agent behavior, as well as accuracy, might be impacted."
